src/demo/DemoTestActivitiesService.java

```diff
@@ -1,5 +1,6 @@
 package demo;
```

Comment:
Could you please add a test method for the `getActivities(Date fromDate, Date toDate, Integer offset, Integer limit)` method, too? This method is public and it can be used by our partners. I think it should be a pretty small test, because the only thing this method does, is to delegate its work to an overloaded version of it. Thanks!